vaas-app/src/vaas/manager/middleware.py

```diff
@@ -88,6 +89,9 @@ def process_response(self, request, response):
 
                     if isinstance(result.result, Exception):
                         raise result.result
+            except SoftTimeLimitExceeded:
+                logging.info("Time for finish the task has been reached: The task with id {} will be killed.".format(
```

Comment:
info? I think this should be logging.error

Reply:
👍